### ISSUE TITLE
What is the status of this plugin?

### ISSUE THREAD

**Issue by fitzage** [KraigM/homebridge-wink#1]
I'm currently having a wink issue with the homebridge-legacy-plugins module, so I'm wondering if I can plug this in instead. Is it ready for that? Also, is it the same one that's here? https://www.npmjs.com/package/homebridge-wink (I notice the usernames aren't the same…)

And if I plug this in, how does the config know to use this instead of the one in homebridge-legacy-plugins? Does it automatically take over?

**Comment by KraigM** [comment]
I started splitting this off but haven't finalized it.  I think it is done, but just need to test it out before making it official.  @pdlove said he was going to take it over, so I was just setting this up in the meantime, but got busy.

**Comment by schmittx** [comment]
Would love to see binary switches added to this...I'm trying to work on it myself but my coding experience is limited...

**Comment by KraigM** [comment]
@pdlove I went to publish this today but it looks like you already did two weeks ago.  Are you wanting to take ownership or should I go ahead and publish mine?

**Comment by schmittx** [comment]
I managed to get binary switches working and confirmed, I used the existing light bulb code as a base and copied/modified accordingly...I can post later this week/weekend once I tidy up my work a bit.

**Comment by pdlove** [comment]
Sorry, I've been out. I though I unpublished it. I've been having a lot of
trouble with my wink hub lately but expect to finish the platform up this
weekend. I have almost all device types reporting details back to
HomeKit... I just need to finish out the issuing of commands.

**Comment by KraigM** [comment]
@pdlove It is still there so I still can't publish. According to the man page, you have to run "npm unpublish homebridge-wink" twice (once to remove the only version out there and again to remove it from the registry entirely)

**Comment by techyowl** [comment]
Just curious if Wink Relay connected devices are not supported or if they will be in the upcoming release. I can try to help I have some programming background. Anyways keep up the awesome work!

**Comment by KraigM** [comment]
@techyowl what are you wanted to be integrated with wink relay? Mainly  is there some information and/or settings on the wink relay that you want to be able to control from HomeKit or are you looking to have wink relay actually control HomeKit?

**Comment by techyowl** [comment]
well I have both buttons on the wink relay controlling my ceiling fans and ceiling lights but they don't show up in the home kit devices on Inteon+ so I got the impression the wink relay isn't support. I did find out recently that it still finds lights connected through the wink relay though such as my GE Link bulb. at least I think that is what happened. also do you have plans to support outlet based wink compatible devices such as the pivot power genius or others? if so that would be awesome!

**Comment by KraigM** [comment]
I don't know enough about the wink relay, but if it can be treated as a switch, then maybe.  But if you are wanting all HomeKit devices to show up on the wink relay screen, thats a much bigger problem.  HomeBridge/HAP work to pretend to be devices that can be controlled by HomeKit, not the other way around.

I know @pdlove and @schmittx have been working on adding support for more devices.  

Side Note for @pdlove and @schmittx: Feel free to send me pull requests.  If you want me to, I can setup a feature branch so you guys can collaborate (I use gitflow style in my repos, hence develop and master).

**Comment by KraigM** [comment]
FYI to all, the initial version of this plugin has been released on npm.  If you have both the legacy plugins and the new separate wink plugin, it gets unhappy.  As soon as @nfarina has a chance to kill off the old version (I will prob go ahead and send him a pull request to make it easier on him) I will close this item.  There is a channel on #slack called #homebridge-wink where you can ask any questions or post any suggestions.  You can also create issues and pull requests in this repo from now on (instead of bugging @nfarina in legacy plugins :-) )

**Comment by techyowl** [comment]
oh I don't expect to be able to control home kit devices from my wink relay I was always referring to making the wink relay buttons home kit compatible.

**Comment by KraigM** [comment]
ok that might work.  You can post that separately as a feature request

**Comment by techyowl** [comment]
I tried joining the slack channel and I couldn't figure out how to join that channel because I'm not on any team that has access to that channel and github only offers issue reporting so I don't know how or where to send a feature request '(X,X)'

**Comment by KraigM** [comment]
You sign up for the HomeBridge slack team. You can have multiple slack accounts and switch between them. 

As far as making the request, create an "issue" and mark it as an enhancement (yeah never been a fan of calling bugs and features both "issues")

**Comment by techyowl** [comment]
ok well I guess slack only allows invite only team registration so I have no way of joining the slack team by going to homebridge.slack.com which is fine and I created the issues but from what I gathered from Githubs helpdesk only the owner of the repo can tag labels to issues, otherwise I would have tagged them. I feel like such a new guy to all this stuff. I'm used to programming on my own so these collaboration tools are new to me.
